configuration/items.md

```diff
@@ -311,9 +324,13 @@ Groups are supported in Sitemaps, Automation Rules and other areas of openHAB.
 A simple example Group definition is:
 
 ```java
-Group Temperatures
+Group gGroundFloorTemperatures
```

Comment:
My suggestion would be to stick with `gTemperatures`.

Reply:
Again, this is an example. An example to show how to name a multi-word Group.

However I should probably give a list of Group name examples, just like above.